### PR TITLE
Code style: PHPCS errors fixes, consitency

### PR DESCRIPTION
**Pull request by vmalyk**
Fixed:
1.  Replace is_null to strict comparison and fixed offset for doc block
2. Removed tabs and 4 spaces.
3. Fixed consistency for constants and used one quote

### CODE REVIEW

File: Model/Config.php [hunk 1]
```diff
@@ -146,7 +146,9 @@ public function getConfig($configPath, $scopeId = null, $scope = null)
         if (!$scope && $this->isSingleStoreMode()) {
             return $this->scopeConfig->getValue($configPath);
         }
-        return $this->scopeConfig->getValue($configPath, $scope ?: ScopeInterface::SCOPE_STORE, is_null($scopeId) ? $this->storeManager->getStore()->getId() : $scopeId);
+        $scopeId = ($scopeId !== null) ?: $this->getCurrentStoreId();
```
Comment (pniel-cohen) [inline Model/Config.php]: This is incorrect & should be:
`$scopeId = ($scopeId === null) ? $this->storeManager->getStore()->getId() : $scopeId`

File: Model/Config.php [hunk 2]
```diff
@@ -313,9 +315,8 @@ public function isActivated($scopeId = null, $scope = null)
      */
     public function setStoreCredentialsAndIsEnabled($appKey, $secret, $isEnabled, $storeId = null, $scopes = ScopeInterface::SCOPE_STORES)
     {
-        if (is_null($storeId)) {
-            $storeId = $this->storeManager->getStore()->getId();
-        }
+        $storeId = ($storeId !== null) ?: $this->storeManager->getStore()->getId();
```
Comment (pniel-cohen) [inline Model/Config.php]: This is incorrect, should stay unchanged or be:
`$storeId = ($storeId === null) ? $this->storeManager->getStore()->getId(); : $storeId`

Comment (vmalyk) [reply Model/Config.php]: Fixed, with small addition - I use $this->getCurrentStoreId().

File: Model/Config.php (hunk 1 above)
Comment (vmalyk) [reply Model/Config.php]: Fixed, with small addition - I use $this->getCurrentStoreId().

Comment (vmalyk) [reply Model/Config.php]: Fixed, with small addition - use $this->getCurrentStoreId().

**Comment by vmalyk** [comment]
@pniel-cohen ,

Thank you for your review. I added suggested fixes.

File: Model/Config.php [hunk 3]
```diff
@@ -146,7 +146,9 @@ public function getConfig($configPath, $scopeId = null, $scope = null)
         if (!$scope && $this->isSingleStoreMode()) {
             return $this->scopeConfig->getValue($configPath);
         }
-        return $this->scopeConfig->getValue($configPath, $scope ?: ScopeInterface::SCOPE_STORE, is_null($scopeId) ? $this->storeManager->getStore()->getId() : $scopeId);
+        $storeId = ($storeId === null) ? $this->getCurrentStoreId() : $storeId;
```
Comment (pniel-cohen) [inline Model/Config.php]: This would lead to an error. Should be:
`$storeId = ($scopeId === null) ? $this->getCurrentStoreId() : $scopeId;`

Comment (vmalyk) [reply Model/Config.php]: @pniel-cohen , Fixed

Comment (vmalyk) [reply Model/Config.php]: @pniel-cohen , it's  fixed.

**Comment by vmalyk** [comment]
@lgreenboim, can you provide what is issues and might be it's not related with current PR?

**Comment by lgreenboim** [comment]
@vmalyk , I am sorry, I got confused with PR:
https://github.com/YotpoLtd/magento2-module-yotpo-reviews/pull/214

**Pull request merged**